src/uct/ib/ud/base/ud_iface.c

```diff
@@ -530,12 +530,12 @@ void uct_ud_iface_query(uct_ud_iface_t *iface, uct_iface_attr_t *iface_attr)
     /* The first iov is reserved for the header */
     iface_attr->cap.am.max_iov        = uct_ib_iface_get_max_iov(&iface->super) - 1;
 
-    iface_attr->cap.put.max_short     = iface->config.max_inline - sizeof(uct_ud_neth_t) - sizeof(uct_ud_put_hdr_t);
+    iface_attr->cap.put.max_short     = iface->config.max_inline -
+                                        sizeof(uct_ud_neth_t) - sizeof(uct_ud_put_hdr_t);
     iface_attr->cap.put.max_bcopy     = 0;
+    iface_attr->cap.put.min_zcopy     = 0;
     iface_attr->cap.put.max_zcopy     = 0;
-    iface_attr->cap.put.max_iov       = 1;
-
-    iface_attr->cap.get.max_iov       = 1;
+    iface_attr->cap.put.max_iov       = 0;
```

Comment:
IMHO we need to remove those anyway since pu/get are unsupported in ud, just need to make sure it's all initialized to 0